module/purpose/Soccer/data/config/Soccer.yaml

```diff
@@ -10,3 +10,6 @@ position: DYNAMIC
 
 # The number of seconds to wait before assuming a teammate is inactive
 timeout: 8
+
+# The largest robot id number
+max_robots: 4
```

Comment:
I was looking a lot at the game controller yesterday and it appears this is a variable sent by gamecontroller, so we could use that here